projects/ngrx.io/content/guide/component-store/effect.md

```diff
@@ -91,3 +91,23 @@ An easy way to handle the response in ComponentStore effects in a safe way, with
     );
   });
 </code-example>
+
+## Calling an `effect` without parameters
+
+A common use case is to call the `effect` method without any parameters. 
+To make this possible:
+- The generic type of the `effect` method needs to be set to `void`.
+- The name of the source stream doesn't matter: `trigger$`, `source$` or `$` are good names.
```

Comment:
I was a bit confused by the second bullet point because it has nothing to do with "to make this possible".

What about:  
```suggestion
To make this possible, set the generic type of the `effect` method to `void`.
```

"The name of the source stream doesn't matter, `trigger$`, `source$` or `$` are good names that are often used. We encourage to choose one of these and them consistently in your codebase." can be a comment in the code block.

Reply:
Yes I agree with you. First I wrote this information on both side, but I have removed it from the code comment. I will revert my change